graphql-dgs-codegen-core/src/test/resources/kotlin2/projectionWithSameDefinitions/expected/DgsClient.kt

```diff
@@ -0,0 +1,27 @@
+package kotlin2.projectionWithSameDefinitions.expected
```

Comment:
I don't think you are comparing the _expected_ classes with the generated.

I do think is nice to observe them this way in the PR but I don't think it should be part of the code we commit. Unless we actually write a test that compares the expected vs the generated.

Reply:
Valuable advice！👍🏻

Reply:
Some have been fixed.